Gruntfile.js

```diff
@@ -43,16 +43,31 @@ module.exports = function (grunt) {
 		sauceLoginFile: grunt.file.exists('SAUCE_API_KEY.yml') ? grunt.file.readYAML('SAUCE_API_KEY.yml') : undefined,
 		sauceUser: process.env.SAUCE_USERNAME || 'fuelux',
 		sauceKey: process.env.SAUCE_ACCESS_KEY ? process.env.SAUCE_ACCESS_KEY : '<%= sauceLoginFile.key %>',
-		allTestUrls: ['2.1.0', '1.11.0', '1.9.1', 'browserGlobals'].map(function (ver) {
-			if (ver === 'browserGlobals') {
-				return 'http://localhost:<%= connect.testServer.options.port %>/test/fuelux-browser-globals.html';
+		// TEST URLS
+		distTestUrl: ['http://localhost:<%= connect.testServer.options.port %>/test/?testdist=true'],
+		allTestUrls: ['2.1.0', '1.11.0', '1.9.1', 'browserGlobals', 'codeCoverage' ].map(function (type) {
+			if (type === 'browserGlobals') {
+				return 'http://localhost:<%= connect.testServer.options.port %>/test/browser-globals.html';
+			}
+			else if (type === 'codeCoverage') {
+				return 'http://localhost:<%= connect.testServer.options.port %>/test/?coverage=true';
+			}
+			else {
+				// test dist with multiple jQuery versions 
+				return '<% distTestUrl %>'; //'http://localhost:<%= connect.testServer.options.port %>/test/?jquery=' + ver + '&testdist=true';
 			}
-
-			return 'http://localhost:<%= connect.testServer.options.port %>/test/fuelux.html?jquery=' + ver + '&testdist=true';
 		}),
-		testUrl: ['http://localhost:<%= connect.testServer.options.port %>/test/fuelux.html?jquery=' + '1.9.1&testdist=true'],
 
 		//Tasks configuration
+		blanket_qunit: {
+			source: {
+				options: {
+					urls: ['http://localhost:<%= connect.testServer.options.port %>/test/?coverage=true&gruntReport'],
+					threshold: 68,
+					globalThreshold: 81
```

Comment:
That's good for `servefast`. However, getting into the habit of actively ignoring errors seems at the least odd and at the most outright dangerous (when I've seen this done before, it usually ends up causing some major problems). I've heard several people say things like "ugh, our grunt setup is useless because I always just `--force` everything". If everyone is always forcing everything anyways, why not just make that the default behaviour and then allow the non-desirable behaviour to be an intentional decision? Might be good to hear others weigh in on this though. @swilliamset @futuremint